spinalcordtoolbox/scripts/sct_register_multimodal.py

```diff
@@ -348,6 +349,7 @@ def main(argv=None):
     else:
         fname_mask = ''
     padding = arguments.z
+    paramregmulti = deepcopy(DEFAULT_PARAMREGMULTI)
```

Comment:
Good fix, but it raises the deeper question of why the tests are modifying their input at all.